src/extension.ts

```diff
@@ -47,26 +50,29 @@ export class Extension {
 
     setEmulatorState(state: EmulatorState) {
         this.emulatorState = state;
+        this.api.changeEmulatorState(state)
+        refreshCodeLenses();
     }
 };
 
 export enum EmulatorState {
-    Stopped = 1,
+    Stopped = 0,
     Starting,
     Started,
 }
 
 // Called when the extension starts up. Reads config, starts the language
 // server, and registers command handlers.
-export function activate(ctx: ExtensionContext) {
+export async function activate(ctx: ExtensionContext) {
     let config: Config;
     let terminal: Terminal;
     let api: LanguageServerAPI;
 
     try {
         config = getConfig();
+        await config.readLocalConfig()
         terminal = createTerminal(ctx);
-        api = new LanguageServerAPI(ctx, config);
+        api = new LanguageServerAPI(ctx, config, EmulatorState.Stopped, {name: "", address: ""});
```

Comment:
I see this empty account object in a few places -- it's quite prone to errors if the format changes. 

```suggestion
        api = new LanguageServerAPI(ctx, config, EmulatorState.Stopped, null);
```

Maybe update the `LanguageServerAPI` constructor like this:

```ts
export class LanguageServerAPI {

  constructor(
    ctx: ExtensionContext, 
    config: Config, 
    emulatorState: EmulatorState, 
    activeAccount: Account | null,
  ) {
    // ...
    const activeAccountName = activeAccount ? activeAccount.name : ""
    const activeAccountAddress = activeAccount ? activeAccount.address: ""
  }